RELEASE-NOTES.md

```diff
@@ -1,4 +1,60 @@
-# PatternFly Albatross Milestone (2019-07-02)
+## 2019.04 release notes (2019-07-24)
+- @patternfly/patternfly@2.23.0
+
+**About modal:** Updated break-word on content area instead of break-all ([#2035](https://github.com/patternfly/patternfly-next/pull/2035))
```

Comment:
@rachael-phillips wants these to have `AutoLinkHeader` anchors on -org, so please make them `h3`s like `### About modal` like I did for the other notes.